packages/engine/src/HedronEngine/HedronEngine.ts

```diff
@@ -160,6 +210,17 @@ export class HedronEngine {
     return stripForSave(this.store.getState())
   }
 
+  /**
+   * Recreates global option nodes for all registered plugins
+   * This should be called after loading a project or resetting the store
+   */
+  public recreateGlobalOptionNodes() {
```

Comment:
on project load, the state gets cleared so recreating the global options. Not sure if we want these to go beyond the project, maybe we just wait to register them until load, but then we need to hide the widget strip until loaded

Reply:
I think I'm a bit confused on what these two methods are doing and why we need it compared to how its working for the LFO input right now, probably worth having a chat soon to discuss it all?